scripts/install-mender.sh

```diff
@@ -152,21 +195,82 @@ add_repo() {
     echo "$repo" > /etc/apt/sources.list.d/mender.list
 }
 
-do_install() {
+do_install_open() {
+    # Filter out commercial components
+    local selected_components_open=""
+    for c in $SELECTED_COMPONENTS
+    do
+        if ! echo "$COMMERCIAL_COMPONENTS" | egrep -q "(^| )$c( |\$)"; then
+            selected_components_open="$selected_components_open $c"
+        fi
+    done
+
+    # Return if no open source components selected
+    if [ -z "$selected_components_open" ]; then
+        return
+    fi
+
+    echo "  Installing open source components from APT repository"
+
     apt-get update
     apt-get install -y \
        -o Dpkg::Options::="--force-confdef" \
        -o Dpkg::Options::="--force-confold" \
-       $SELECTED_COMPONENTS
+       $selected_components_open
+
+    echo "  Success! Please run \`mender setup\` to configure the client."
+}
+
+do_install_commercial() {
+    # Filter commercial components
+    local selected_components_commercial=""
+    for c in $SELECTED_COMPONENTS
+    do
```

Comment:
I do not think we format shell code with do in a separate line.

Reply:
I know. I based this syntax on the existing code... but once you comment I'll fix the old code too.